### PR TITLE
User profile 

### PR DESCRIPTION
**Pull request by SukanyaSen530**
- profile page

### CODE REVIEW

**Review by rutvikpumak — APPROVED** [review]
Code look good and well structured
Goot to merge

File: src/pages/User/UserProfile.jsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        <p>
+          <span>Email </span> <span> : {details?.email} </span>
+        </p>
+        <p>
+          <span>Joined </span>{" "}
+          <span>
+            :{" "}
+            {`${new Date(details?.createdAt).toLocaleString("default", {
+              month: "long",
+            })} ${new Date(details?.createdAt).getFullYear()}`}
```
Comment (dev-enforced) [inline src/pages/User/UserProfile.jsx:34]: Good way of displaying Joining Date!!

**Review by Richa-rani02 — APPROVED** [review]
Code look good easily understandable

**Review by dev-enforced — APPROVED** [review]
The live preview is working well!!!
Tried the guest login part and viewed the profile tab it's looking good
image

Good work @SukanyaSen530

**Pull request merged**